server/src/main/java/org/candlepin/model/CertificateSerialCurator.java

```diff
@@ -70,7 +62,7 @@ public CertificateSerialCurator() {
 
         DetachedCriteria criteria = DetachedCriteria.forClass(CertificateSerial.class)
             .add(Restrictions.le("expiration", Util.yesterday()))
-            .add(getRevokedCriteria());
+            .add(Restrictions.eq("revoked", true));
```

Comment:
This will delete those that have expired AND been marked as revoked. Is that the intention?